appengine/datastore/src/test/java/com/example/appengine/QueriesTest.java

```diff
@@ -694,15 +694,15 @@ public void queryRestrictions_surprisingMultipleValuesAllMustMatch_returnsNoEnti
     // [START surprising_behavior_example_1]
```

Comment:
Fine as is,  though a shorter alternative could be:
`List<Long> xs = Arrays.asList(1L, 2L);`